plugins/vpp/ifplugin/descriptor/interface_crud.go

```diff
@@ -153,6 +153,26 @@ func (d *InterfaceDescriptor) Create(key string, intf *interfaces.Interface) (me
 			d.log.Error(err)
 			return nil, err
 		}
+
+	case interfaces.Interface_GTPU_TUNNEL:
+		var multicastIfIdx uint32
+		multicastIf := intf.GetGtpu().GetMulticast()
+		if multicastIf != "" {
+			multicastMeta, found := d.intfIndex.LookupByName(multicastIf)
+			if !found {
+				err = errors.Errorf("failed to find multicast interface %s referenced by GTPU %s",
+					multicastIf, intf.Name)
+				d.log.Error(err)
```

Comment:
I doubt if we should log the error. The caller receives the error and she could do whatever works best for her, say logging.

Reply:
I was following the convention of the existing implementation. The error is logged in all the other cases as well.